packages/core/http/core-http-server-internal/src/http_config.ts

```diff
@@ -167,6 +167,14 @@ const configSchema = schema.object(
       }
     ),
     restrictInternalApis: schema.boolean({ defaultValue: false }), // allow access to internal routes by default to prevent breaking changes in current offerings
+    versioned: schema.object({
+      // Which handler resolution algo to use: "newest" or "oldest"
+      handlerResolution: schema.oneOf([schema.literal('newest'), schema.literal('oldest')], {
+        defaultValue: 'oldest',
+      }),
```

Comment:
NIT: Handler resolution or version resolution? Is the "oldest handler" really a valid term?

Reply:
Will change to `versionResolution`